release-schema.json

```diff
@@ -63,6 +63,16 @@
           "codelist": "methodOfPayment.csv",
           "openCodelist": true,
           "uniqueItems": true
+        },
+        "payee": {
+          "title": "Payee",
+          "description": "The organization that receives the participation fees.",
+          "$ref": "#/definitions/OrganizationReference"
+        },
+        "paymentAddress": {
+          "title": "Payment address",
```

Comment:
For consistency, maybe re-title `methodOfPayment` from "Method(s) of payment" to "Payment methods".